First Pull Portfolio: setup and mobile version skeleton.

abelagza opened the pull request:
## In this Milestone I:

- Created REPO
- Created New Branch and Files
- Updated HTML&CSS files
- Added Linters
- Fixed linter errors locally
- Added Final touches

abelagza committed:
add HTML&CSS files

abelagza committed:
changes on the html&css files

abelagza committed:
add images and icons

abelagza committed:
Update HTP&CSS file

abelagza committed:
Add changes to HTML&CSS files

abelagza committed:
Add Linters

abelagza committed:
Update gitignore file

abelagza committed:
Fix linter errors

abelagza committed:
Update the README file

abelagza committed:
Update the README

abelagza committed:
update readme

abelagza committed:
update readme

abelagza committed:
fix linter error

elmar8287 commented on .github/workflows/linters.yml:
```diff
@@ -0,0 +1,62 @@
+          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.eslintrc.json
+      - name: ESLint Report
+        run: npx eslint .
```
- [ ] Please, consider removing the line for JS, as it is not used in this milestone 👍 By doing this, your linter issues will be solved.

elmar8287 commented on README.md:
```diff
@@ -2,26 +2,21 @@
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
```
- [ ] Please, consider removing the not used section and also adding the instruction on how to set up your project on the local machine, using `git clone` command. It will help to set up your project for other developers.

elmar8287 commented on index.html:
```diff
@@ -0,0 +1,71 @@
+
+<body>
+
+<!-- header section -->
```
- [ ] Please, consider removing the comments to make your code more clear 👍

elmar8287 commented on index.html:
```diff
@@ -0,0 +1,71 @@
+        </div>
+    </div>
+
+    </section>
```
- [ ] Please, consider putting this toolbar section into the `header` tag, as required in the project. 
- [ ] Please, consider putting the hamburger icon into the `nav` tag, as required in the project.

elmar8287 commented on index.html:
```diff
@@ -0,0 +1,71 @@
+        <div class="container">
+
+            <div class="left">
+                <h1 class="title">Abel</h1>
+                <h1 class="title">Aguilar</h1>
```
- [ ] Please, consider that you have to use the `h1` tag one time in your code. The `h1` is the main header, which talks to the browser what about is your page.

elmar8287 commented on index.html:
```diff
@@ -0,0 +1,71 @@
+                <p class="text">Hello! I am a software developer! I can help you build a product,  
+                <p class="text">Take a look of my works.</p>
+                <p class="text">
+                    If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
```
- [ ] I think it will be better to use the `p` tag one time and insert the whole paragraph 👍

elmar8287 commented on index.html:
```diff
@@ -0,0 +1,71 @@
+        </div>
+
+    </section>
+<br><br><br><br><br><br><br><br><br><br><br>
```
- [ ] It will be better to not use `br` tag and try to fix styling in .css file 👍

elmar8287 commented on index.html:
```diff
@@ -0,0 +1,71 @@
+           <a href="#"><img src="./icons/linkedin.png" alt="" class="linkedin"></a>
+        </div>
+
+    </section>
```
- [ ] Please, consider that you do not need to implement the footer section on this milestone 👍

elmar8287 reviewed (changes requested):
Hi, @abelagza. 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Check the comments under the review.
- [ ] Please, consider fixing the position of the hamburger icon - it is aligned to the center.

> You version
<img width="290" alt="Screen Shot 2022-02-18 at 01 03 56" src="https://user-images.githubusercontent.com/49064106/154570927-c6b1a85e-f0b4-45b5-8a3a-3aa07472e7cf.png">

- [ ] Please, consider using the required fonts for header `Playfair Display` and for paragraph and button `Poppins`
- [ ] Please, consider that the branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section')

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

abelagza committed:
Fix issues

abelagza committed:
Fix errors

abelagza committed:
Add require changes

abelagza committed:
Add linter changes

abelagza committed:
Add changes

abelagza committed:
Add changes

abelagza commented:
## In this Milestone I:

- Created REPO
- Created New Branch and Files
- Updated HTML&CSS files
- Added Linters
- Fixed linter errors locally
- Added Final touches

**Requested review**

- made review suggested changes

elmar8287 commented:
@abelagza you have linter issues, but if you go "Details", you can see that they are not difficult to solve. Please, fix them and submit them again:
<img width="981" alt="Screen Shot 2022-02-18 at 13 46 05" src="https://user-images.githubusercontent.com/49064106/154658607-5f1fb5c0-d09a-47f3-8393-9923c68eb975.png">

If you have any questions, I am here for assistance.

abelagza committed:
Make required changes

abelagza committed:
Make requested changes

Donard97 commented on index.html:
```diff
@@ -0,0 +1,56 @@
+    <section id="body">
+        <div class="container">
+
+            <div class="left">
+                <h1 class="title">Abel</h1>
+                <h1 class="title">Aguilar</h1>
```
- [ ] Good job on implementing other requested changes but as previous reviewer said kindly consider to use the h1 tag only one time for example in this situation you can put Abel Aguilar inside of only one h1. Keep rocking 💪

Donard97 commented on style.css:
```diff
@@ -0,0 +1,293 @@
+  width: 100%;
+}
+
+#body .container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```
- [ ] You did a tremendous work on designing the first section of you portfolio but please stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) as it was part of [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md).

Here you have a screenshot of your project:

![image](https://user-images.githubusercontent.com/74506933/154732073-7d5c5929-9979-46bf-9a51-69d1b8dc13ba.png)

Here you have a screenshot of the design:

![image](https://user-images.githubusercontent.com/74506933/154732184-dbbe4452-cc14-41bc-ad6e-4a9f2cf1bc79.png)

Donard97 reviewed (changes requested):
## Hi @abelagza , 👋 
    
Great work on making the changes requested by a previous reviewer 👏 💯 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
    
## Required Changes ♻️

- [ ]  As the previous reviewer requested please, consider that the branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'). You can use this [link](https://linuxize.com/post/how-to-rename-local-and-remote-git-branch/) for guidance in the future when you want to rename the branch.
- [ ] Please check the comments under the review.
    
 ## Optional suggestions
    
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
    
**Cheers and Happy coding!👏👏👏**
    
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
    
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
    
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
 ```
